python/ledger_tutorial.py

```diff
@@ -42,9 +42,5 @@
             # Ledger verification is happening implicitly in ccf.ledger.Ledger()
             for key, value in public_tables[target_table].items():
                 target_table_changes += 1  # A key was changed
-                # Log the key and value for the transaction on the target table
-                # The target_table: 'public:ccf.gov.nodes.info' has already been decoded in ledger.py
-                # For other tables knowledge of serialization scheme used is important.
-                # If the table was using msgpack, use ccf.ledger.extract_msgpacked_data(data)
```

Comment:
Can we restore some of this comment? Is the logging line below showing something interesting because the target table is JSON-to-JSON, or just dumping some binary? We should still say that serialization schemes are per-table and must be known to interpret anything useful here.

Reply:
Good point. I've added a comment to explain this. Both `key` and `value` are raw bytes, and it's up to this code to know how to deserialise them.